uitests/issue9553/readme.md

```diff
@@ -0,0 +1,5 @@
+# issue9553
+
+undefined
+
+undefined
```

Comment:
Is this file supposed to be committed as it is?

Reply:
oops, thats a bug in the skeleton generation